client/my-sites/posts/post-share.jsx

```diff
@@ -212,6 +226,7 @@ export default connect(
 		const userId = getCurrentUserId( state );
 
 		return {
+			planHasRepublicizeFeature: hasFeature( state, siteId, FEATURE_REPUBLICIZE ),
```

Comment:
> Do we really want to start using client side only checks

We do it all over already for all the nudges, this is nothing new. The backend side system is really confusing and is missing plenty granularity to control all the features

> What happens when we decide to show only a few items?

I put some thought into that and we need to put "showonplanspage" property for the features that defaiults to true.